### PR TITLE
mixer deleted but quadx

### PR DESCRIPTION
**Pull request by budebulai**
flight mode not modified yet

### CODE REVIEW

File: src/main/config/config.c [hunk 1]
```diff
@@ -759,9 +656,6 @@ void validateAndFixConfig(void)
         if (masterConfig.batteryConfig.currentMeterType == CURRENT_SENSOR_ADC) {
             featureClear(FEATURE_CURRENT_METER);
         }
-#endif
```
Comment (Innoecho) [inline src/main/config/config.c:762]: defined(STM32F10X) is TRUE

File: src/main/io/serial_msp.c [hunk 2]
```diff
@@ -1934,4 +1923,4 @@ void showCurrentPortInfo(void)
     printf("currentPort->checksum:%d\r\n",currentPort->checksum);
     printf("currentPort->cmdMSP:%d\r\n",currentPort->cmdMSP);
     printf("currentPort->inBuf:%s\r\n",currentPort->inBuf);
-}
\ No newline at end of file
+}
```
Comment (Innoecho) [inline src/main/io/serial_msp.c:1926]: keep a new line at end of file

File: src/main/rx/rx.c [hunk 3]
```diff
@@ -153,7 +153,7 @@ void rxInit(rxConfig_t *rxConfig)
     }
 
     if (!feature(FEATURE_3D)) {
-        rcData[0] = rxConfig->rx_min_usec;
+        rcData[0] = rxConfig->rx_min_usec;  //wrong,why not rcData[3]???
```
Comment (Innoecho) [inline src/main/rx/rx.c:156]: rcData[THROTTLE]?

Comment (budebulai) [inline src/main/rx/rx.c:156]: Yeah, if enable 3D feature, channel throttle's pwm is initialised midrc, 1500. so this place should reset rcData[3] to rx_min_usec. do you think so ?

File: src/main/config/config.c (hunk 1 above)
Comment (budebulai) [inline src/main/config/config.c:762]: yes, STM32F10X is defined in makefile

File: src/main/rx/rx.c (hunk 3 above)
Comment (Innoecho) [inline src/main/rx/rx.c:156]: Yes

File: src/main/config/config.c (hunk 1 above)
Comment (Innoecho) [inline src/main/config/config.c:762]: so, why persist #if defined(STM32F10X)

**Comment by CCharlieLi** [comment]
@Innoecho I wonder if it works fine with board. Could you have a test?

**Comment by budebulai** [comment]
Yeah, I've upload simplified firmware on Naze32, and tested quadx. Maybe we should fly it a time.

**Comment by CCharlieLi** [comment]
Thanks, Merged. Next time, we should build a workflow to test our project.

**Pull request merged**